adding cran maintainers 2017 info from Pinto et al. 2018

annnvv opened the pull request.

hturner commented on content/data.md:
```diff
@@ -8,7 +8,9 @@ tocify: true
 ### Current Efforts
 
-The survey sub-team is focused on monitoring diversity and exploring obstacles, motivations, and involvement of useRs in the community.  Diversity will be monitored via basic demographic summaries from specifically designed surveys or other community surveys (e.g. useR! feedback survey).
+The survey sub-team is focused on monitoring diversity and exploring obstacles, motivations, and involvement of useRs in the community.  
```
You've got two spaces at the end of this line, so the "Diversity..." sentence starts on a new line. Either remove one of the spaces so you just start a new sentence, or add a line break to start a new paragraph.

annnvv replied:
Incorporated in 56e8927

hturner commented on content/data.md:
```diff
@@ -19,6 +21,53 @@ giving a rough approximation that assumes binary gender categories.
 
```
We'd better update this a little too. Eventually it can go, but still applies to the useR! stats, so let's put it in past tense:
 
> The taskforce collated data on the gender breakdown and other demographics of
contributors to the R project, particularly R Foundation supported activities. 
For many of the gender summaries the gender was predicted based on first name,
giving a rough approximation that assumes binary gender categories.

annnvv replied:
Incorporated in 56e8927

hturner commented on content/data.md:
```diff
@@ -19,6 +21,53 @@ giving a rough approximation that assumes binary gender categories.
+scientist. The survey was conducted starting in late November 2017 and ending in early January 2018. The respondents were contacted 
+by email and asked to complete an online questionnaire, with a response rate of 24% (a total of 1,553 responses). 
+
+<div class="commentbox"> In 2017, 10.9% of package maintainers who identified as scientists were women. </div>
```
Can we update this to women and non-binary, since we have the data.

annnvv replied:
Incorporated in 56e8927

hturner commented on content/data.md:
```diff
@@ -19,6 +21,53 @@ giving a rough approximation that assumes binary gender categories.
 
 ## Packages
 
+### Maintainer Gender 2017
```
Let's say "Maintainer Demographics 2017". The focus on gender is historical (as we were originally a taskforce on women in R).

annnvv replied:
Incorporated in 56e8927

hturner commented on content/data.md:
```diff
@@ -19,6 +21,53 @@ giving a rough approximation that assumes binary gender categories.
+
+- Overwhelmingly, 80.0% of respondents identified as either having completed or are in progress of completing a doctorate degree. 
+
+- Almost half (48.9%) of the respondents identify as being located in Europe and another 34.1% as being located in North America.
```
May we could also highlight the statistic "x% of respondents were located outside Europe and North America". This could be in addition or instead of this bullet point.

annnvv replied:
Incorporated in 910a587

hturner commented on content/data.md:
```diff
@@ -19,6 +21,53 @@ giving a rough approximation that assumes binary gender categories.
+
+<div style="display: flex; justify-content: center">
+<img src="../images/pinto_et_al_2018_gender_plot.png" 
+    alt="In 2017, of the package author surveyed, 10.9% of respondents self-identify as female, about 88.2% identify as male, 0.1% of respondents self-identify as non-binary."
```
author -> authors

self-identify -> self-identified

identify -> identified

(maybe be consistent about using identify/self-identify?)

annnvv replied:
Incorporated in 56e8927 (chose self-identified to match the plots)

hturner commented on content/data.md:
```diff
@@ -19,6 +21,53 @@ giving a rough approximation that assumes binary gender categories.
+
+<div style="display: flex; justify-content: center">
+<img src="../images/pinto_et_al_2018_educ_plot.png" 
+    alt="80.0% of respondents identify as either having completed or are in 
```
having completed or are in progress of completing a doctorate degree -> having a completed or in-progress doctorate degree.

annnvv replied:
Incorporated in 56e8927

hturner commented on content/data.md:
```diff
@@ -19,6 +21,53 @@ giving a rough approximation that assumes binary gender categories.
+
+- 44.6% of respondents identify between 30 and 40 years of age at the time of the survey.
+
+- Overwhelmingly, 80.0% of respondents identified as either having completed or are in progress of completing a doctorate degree. 
```
having completed or are in progress of completing a doctorate degree -> having a completed or in-progress doctorate degree.

annnvv replied:
Incorporated in 56e8927

hturner commented on content/data.md:
```diff
@@ -19,6 +21,53 @@ giving a rough approximation that assumes binary gender categories.
+    width = "425"/>
+</div>  
+
 ### Maintainer Gender 2016
```
This section can be deleted now.

annnvv replied:
Incorporated in 56e8927

annnvv commented:
Heather, I have incorporated your comments in commit 56e8927 and 910a587. I chose to use self-identified throughout. Also, I updated the plots because I realized by truncating the values, they weren't adding up to 100%. The figures in the new plots are rounded (and I also updated the text to match). 

I made a mistake merging the master branch into this branch, hopefully that doesn't pose a big problem.

annnvv commented:
I took a brief look at the Mair et al. 2010 paper. The authors of the paper use the term authors to refer to the respondents of their email survey. The methodology note says "In total, we had 4,274 email addresses of R package authors. ... The platforms we used for the acquisition of the email addresses were CRAN, R-Forge, and Bioconductor. ... Note that if packages had multiple authors, emails were sent out to those who provided an email address in the package description file. In addition, in the email list we used, some package authors had multiple email addresses. Therefore, the response rate below reflects a lower bound."

I am inclined to use the language that the paper authors use, but can change if maintainers is the more accurate term.

hturner commented:
Thanks, your reasoning makes sense to me, merged in now.
